adds loadout beacons for security and some new toys to support the new equipment spread

kittysmooch opened the pull request:
## About The Pull Request

this pr adds a loadout beacon similar to the captain's signature weapon beacon for security officers to pick a more tightly themed loadout for their character. the current (work in progress) types are:

- the jitte and stunbaton belt pack, as usual
- the sword and stunbaton belt
- a ballistic option that comes with a new stingball launcher and a new relatively low power, disposable personal defense weapon
- maybe a support or medic pack

### the stingball launcher
<img width="189" height="192" alt="image" src="https://github.com/user-attachments/assets/f51fa175-43ae-4759-ba7d-37c82fecfd94" />

a pneumatic stingball launcher that can be fed from its central hopper. the stingballs are relatively low power, but embed on unarmored targets, making this a similar but slightly weaker or sidegraded variation to the disabler.

originally i was gonna have this thing suck air tanks to power the pneumatics, but powerfist/pneumatic cannon code is snowflaked and just about difficult enough to get working with the gun mechanics in a sane way that it was firmly not worth it. plus its just a secondary ammo mechanic which is ZzzzZZZz

### the schießenmaschine
<img width="193" height="191" alt="image" src="https://github.com/user-attachments/assets/ac71f276-33ad-4c29-b6fb-6aa6f63bfd5e" />

its german for shooting machine and definitely has no innuendo. the sprite is gags enabled meaning your replacements can come out of the vending machine in any color under the sun. you will need replacements, because it's a single-use plastic firearm.

this one is the primary reason the pr is still a work in progress, because it's still pretty buggy, with broken inhands and strangely persistent reloadability lol

## Why It's Good For The Game

the original intent was just to offer some loadout variation because we have some security characters that are gun toting types who look kinda silly with the sheathe on their hips. technically they could get a gun and a chest rig from cargo and vendors but then they have to take some time out of their rp plans to get the funds. every round.

the stingball launcher was an idea juni had to slightly nerf the disabler while still giving security the ability to take down the targets tg build the disabler around. specifically, where the disabler can be kinda oppressive to your average spaceman who wasn't wise enough to stock pump up or another stimulant, the stingball launcher is severely weak to armor because a lot of its strength comes from stingball embeds. plus theres no extra stun or anything, just the eventual stamcrit. lastly, it fulfills a design goal of making less lethal options truly less lethal, by making you have to do a decent amount of brute on the way to stamcrit.

the schiebenmaschine guns came out of us kind of only having the det's revolver with a nice paxil sprite, which would have been lame to give to all of sec, and then a bunch of space loot or antag powered pistols. the schiebenmaschine is balanced around having about as much damage in the magazine as other guns in the system, but having really lousy per-shot damage and no AP, making them terrible against armor, and with a severely negative wound bonus to boot. It's further balanced by the fact that, while you can carry a brace of them, they're gun-sized, and probably bigger than e.g. a speedloader or magazine.

## Changelog

:cl:
:cl:

kittysmooch commented on modular_doppler/modular_cosmetics/code/uniform_overrides.dm:
```diff
@@ -26,3 +26,9 @@
 	mask_type = /obj/item/clothing/mask/neck_gaiter
+
+/datum/outfit/job/security
+	suit_store = null
+	backpack_contents = list(
+		/obj/item/signature_beacon/security_equipment_package = 1,
+	)
```
fixed!

00-Steven reviewed (changes requested):
first pass-through, the nonmodular edits especially feel kinda awkward and could probably be done a better way

00-Steven commented:
As for the tests, try updating from master, re-running, and committing the auto-genned map icon dmis

kittysmooch commented on modular_doppler/modular_weapons/code/guns/smg.dm:
```diff
@@ -53,3 +53,61 @@
+	greyscale_config_inhand_left = /datum/greyscale_config/schiebenmaschine_lefthand
+	greyscale_config_inhand_right = /datum/greyscale_config/schiebenmaschine_righthand
+	greyscale_colors = "#bb2222" //randomized on init but we need this for mapping icons and stuff
+	unique_reskin_changes_base_icon_state = FALSE
```
probably leftover

kittysmooch commented:
new changes:

schiebenmaschine magazine now holds 25 rounds, down from 30, for a total magazine potential damage of 250
the fire rate has been halved
added spread, reduced default dual wield spread value to be a little less punishing
removed the schiebenmaschine from the roundstart loadout, retained it in the sec vendor

also fixed the escarabajo icon issue

kittysmooch commented:
a few more tweaks:

- reduced schiebenmaschine price, since it got nerfed
- added a security supply pack for 3 schiebens
- suit slottable-ized the shield. seems harmless given that you can already get a telescopic pretty freely.

kittysmooch commented:
- removed the sword belts from security officer lockers, except the hos and warden since they dont get beacons
- added a new bonus to the gunnery kit, in the form of a pamphlet that teaches you how to throw guns really hard
